### PR TITLE
Gallery boxes moved out of WikibasePageObject

### PR DESCRIPTION
**Pull request by ludwikkazmierczak**
- moved gallery boxes `open` methods out of WikiBasePageObject
- GlobalNavigationPageObject renamed to just GloobalNavigation and cleanup done
- cleaned up code and tests

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/galleryboxes/SpecialMostLinkedFilesPage.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+import com.wikia.webdriver.common.logging.PageObjectLogging;
+
+public class SpecialMostLinkedFilesPage extends GalleryBox {
+
+  public static final String SPECIAL_MOST_LINKED_FILES_PATH = "Special:MostLinkedFiles";
+
+  public SpecialMostLinkedFilesPage open() {
+    getUrl(urlBuilder.getUrlForPath(SPECIAL_MOST_LINKED_FILES_PATH));
+    PageObjectLogging.log("Special Mos tLinked Files Page",
```
Comment (idradm) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/galleryboxes/SpecialMostLinkedFilesPage.java]: Log message misspelled here

**Comment by idradm** [comment]
Shouldn't ``galleryBox`` be a component reused in all of those special pages?

**Comment by ludwikkazmierczak** [comment]
@idradm Not really, component is a part of UI, this class just holds two methods same for all this classes. But we can make it abstract for people not to use a global class ;) And finalize `Page`s, so we people don't extend them.

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/galleryboxes/SpecialUncategorizedFilesPage.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
+package com.wikia.webdriver.pageobjectsfactory.pageobject.special.galleryboxes;
+
+import com.wikia.webdriver.common.logging.PageObjectLogging;
+
+final public class SpecialUncategorizedFilesPage extends GalleryBox {
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/galleryboxes/SpecialUncategorizedFilesPage.java]: MINOR Reorder the modifiers to comply with the Java Language Specification. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AModifiersOrderCheck)

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/galleryboxes/SpecialUnusedVideosPage.java [hunk 3]
```diff
@@ -0,0 +1,15 @@
+package com.wikia.webdriver.pageobjectsfactory.pageobject.special.galleryboxes;
+
+import com.wikia.webdriver.common.logging.PageObjectLogging;
+
+final public class SpecialUnusedVideosPage extends GalleryBox {
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/galleryboxes/SpecialUnusedVideosPage.java]: MINOR Reorder the modifiers to comply with the Java Language Specification. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AModifiersOrderCheck)

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/galleryboxes/SpecialUnusedFilesPage.java [hunk 4]
```diff
@@ -0,0 +1,15 @@
+package com.wikia.webdriver.pageobjectsfactory.pageobject.special.galleryboxes;
+
+import com.wikia.webdriver.common.logging.PageObjectLogging;
+
+final public class SpecialUnusedFilesPage extends GalleryBox {
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/galleryboxes/SpecialUnusedFilesPage.java]: MINOR Reorder the modifiers to comply with the Java Language Specification. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AModifiersOrderCheck)

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/galleryboxes/SpecialMostLinkedFilesPage.java [hunk 5]
```diff
@@ -0,0 +1,16 @@
+package com.wikia.webdriver.pageobjectsfactory.pageobject.special.galleryboxes;
+
+import com.wikia.webdriver.common.logging.PageObjectLogging;
+
+final public class SpecialMostLinkedFilesPage extends GalleryBox {
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/galleryboxes/SpecialMostLinkedFilesPage.java]: MINOR Reorder the modifiers to comply with the Java Language Specification. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AModifiersOrderCheck)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 4 issues:
* MINOR 4 minor

Watch the comments in this conversation to review them.

**Comment by idradm** [comment]
@ludwikkazmierczak so we could as well make it a helper class instead of extending those. What do you think about that?

**Comment by idradm** [comment]
LGTM

**Pull request merged**